cardano-node/src/Cardano/Tracing/Tracers.hs

```diff
@@ -1482,23 +1504,30 @@ data ChainInformation = ChainInformation
     -- current chain.
   , fork :: Bool
     -- ^ Was this a fork.
+  , tipBlockHash :: HeaderHash blk
```

Comment:
I would argue lets not parameterize `ChainInformation` and instead have the following fields:
```
...
  , tipBlockHash :: Text
  , tipBlockParentHash :: Text
...
```
These are chain hashes that eventually get rendered anyways.

Reply:
Done!

Reply:
Done!